### PR TITLE
Amendments #5-7

### PR DESCRIPTION
**Pull request by thatging3rkid**
On April 21st, 2020, amendments 5-7 (Miscellaneous Changes \#1, Changes to Decision Making Process, Addition of Diminished Club State) were formally proposed to the club at-large.

### CODE REVIEW

**Comment by thatging3rkid** [comment]
Draft amendment proposal v1.0: 
[amendment.pdf](https://github.com/RITFIRSTRobotics/constitution/files/4507656/amendment.pdf)

**Comment by thatging3rkid** [comment]
Draft amendment proposal v1.1:
[amendment.pdf](https://github.com/RITFIRSTRobotics/constitution/files/4513415/amendment.pdf)

**Review by ahkneipp — APPROVED** [review]
Looks good to me

**Comment by thatging3rkid** [comment]
Voted and passed on April 28th/29th 2020, awaiting advisor approval

**Comment by thatging3rkid** [comment]
Had to add some things to be compliant with the Fall 2019 Center for Campus Life constitution requirements, should be all set now, just waiting for a review then I'll merge it.

File: constitution.tex [hunk 1]
```diff
@@ -479,6 +482,35 @@ \subsection{Acceptance of Amendments} \label{sect:amending:acceptance}
...
+A person is guilty of hazing in the first degree when, in the course of another person's initiation into or affiliation with any organization, he intentionally or recklessly engages in conduct which creates a substantial risk of physical injury to such other person or a third person and thereby causes such injury.
+Hazing in the first degree is a class A misdemeanor. \\
+
+\noindent \textsection 120.17 Hazing in the second degree.
+A person is guilty of hazing in the second degree when, in the course of another person's initiation or affiliation with any organization, he intentionally or recklessly engages in conduct which creates a substantial risk of physical injury to such other person or a third person.
+Hazing in the second degree is a violation. 
+
+\subsection{Anti-Discrimination Clause} \label{sect:university-policy:anti-discrimination}
+
+This organization shall not discriminate on the basis of sex, race, color, sexual orientation gender identity and gender expression, religion, age marital sate, national origin, disability or veteran status.
```
Comment (ahkneipp) [inline constitution.tex]: Typo

Comment (ahkneipp) [inline constitution.tex]: Should have an oxford comma

**Review by ahkneipp — COMMENTED** [review]
Typos and grammar

File: constitution.tex (hunk 1 above)
Comment (thatging3rkid) [reply constitution.tex]: Both these grammar issues are from the CCL documents that said to copy verbatim, so...

Fixed them anyways, should be all set.

**Review by ahkneipp — APPROVED** [review]

**Pull request merged**